packages/io/svg-deserializer/index.js

```diff
@@ -67,7 +68,7 @@ const instantiate = (src, options) => {
 
   options && options.statusCallback && options.statusCallback({ progress: 50 })
 
-  const result = objectify({ target }, svgObj)
+  const result = objectify({ target, segments }, svgObj)
```

Comment:
Let's change to pass all 'options'
`objectify(options, svgObj)`

I forsee additional options in the future.